invokeai/frontend/web/src/features/gallery/components/ImageViewer/ViewerButton.tsx

```diff
@@ -1,16 +1,33 @@
+import { IconButton } from '@chakra-ui/react';
 import { Button } from '@invoke-ai/ui-library';
 import { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
+import { PiArrowsDownUpBold } from 'react-icons/pi';
 
 import { useImageViewer } from './useImageViewer';
 
 export const ViewerButton = () => {
   const { t } = useTranslation();
   const { onOpen } = useImageViewer();
-  const tooltip = useMemo(() => t('gallery.switchTo', { tab: t('common.viewer') }), [t]);
+
+  const tooltip = useMemo(
+    () =>
+      t('gallery.switchTo', {
+        tab: t('common.viewer'),
+      }),
+    [t]
+  );
 
   return (
-    <Button aria-label={tooltip} tooltip={tooltip} onClick={onOpen} variant="ghost" pointerEvents="auto">
+    <Button
+      aria-label={tooltip}
+      tooltip={tooltip}
+      onClick={onOpen}
+      variant="outline"
+      pointerEvents="auto"
+      sx={{ display: 'flex', gap: 2 }}
+    >
+      <IconButton aria-label={tooltip} variant="ghost" size="sm" icon={<PiArrowsDownUpBold />} />
```

Comment:
Chakra handles this for you

```suggestion
    <Button aria-label={tooltip} tooltip={tooltip} onClick={onClose} variant="outline" leftIcon={<PiArrowsDownUpBold />}>
```